uibackend/server/dashboard_findings_impact.go

```diff
@@ -477,8 +470,28 @@ func (s *ServerImpl) getFindingToAssetCountMapWithFilter(ctx context.Context, fi
 
 		findings := *f.Items
 		log.Debugf("Got findings %+v", findings)
-		if err = processFindings(findings, findingAssetMap, findingToAssetCount); err != nil {
-			return nil, fmt.Errorf("failed to process findings: %w", err)
+
+		for idx, finding := range findings {
+			assetCount := 0
```

Comment:
When I was testing I only saw zeros in the `Affected Assets` column of the Riskiest Assets widget, is this a UI related issue or the actual count should have been visible there?

Reply:
looking into this now 👀

Reply:
Nice, it seems to be fixed now! 🔥